ui/src/main/webapp/home/index.js

```diff
@@ -113,31 +66,36 @@ const LdapTileView = (props) => {
     baseUserDn,
     onDeleteConfig,
     ldapUseCase,
-    attributeMappings
+    attributeMappings,
+    submitting
   } = props
 
   return (
-    <Paper className={styles.config}>
-      <div className={styles.tileTitle}>{ldapUseCase === 'authentication' ? 'LDAP Authentication Source' : 'LDAP Attribute Store'}</div>
-      <ConfigField fieldName='Hostname' value={hostName} />
-      <ConfigField fieldName='Port' value={port} />
-      <ConfigField fieldName='Encryption Method' value={encryptionMethod} />
-      <ConfigField fieldName='Bind User' value={bindUser} />
-      <ConfigField fieldName='Bind User Password' value='******' />
-      <ConfigField fieldName='UserName Attribute' value={userNameAttribute} />
-      <ConfigField fieldName='Base Group DN' value={baseGroupDn} />
-      <ConfigField fieldName='Base User DN' value={baseUserDn} />
-      { ldapUseCase !== 'authentication'
-        ? (
-          <MapDisplay label='Attribute Mappings'
-            mapping={attributeMappings} />
-      ) : null }
-      <RaisedButton style={{marginTop: 20}} label='Delete' secondary onClick={onDeleteConfig} />
+    <Paper className={styles.outerSpinner}>
+      <Spinner submitting={submitting}>
+        <div className={styles.innerSpinner}>
```

Comment:
If it's just strictly the fact that the homepage shouldn't have LDAP/Sources specific information in it, it might be possible to move the component into some sort of "HomePageComponent.js" under each wizard directory that the Homepage just imports and renders. The homepage would still have to know to import and render each wizard's "home page" component though so I'm not sure how much benefit that would have.